### DISCUSSION TITLE
Multiline arg

### DISCUSSION THREAD

**Discussion by hakubo** [johnlindquist/kit#191] [Ideas]
It could be useful in certain applications to support multiline arg.

Having this available could mean we could write simple calculs.
This script https://github.com/johnlindquist/kit/discussions/189 uses 'bc'  under the hood which can be programmed to support more advanced calculations.

**Comment by johnlindquist** [comment]
Hmm 🤔, that makes sense. I'll look into some options. Thanks for the idea!

**Reply by johnlindquist** [reply]
@hakubo Is there an example online somewhere of a multiline/textarea input that fits your needs? A reference I could research would be great to see.
